doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
-|------------------------|--------------------------|-------------------------|------------------|----------------------|
-| .tiny                  | 1                        | 512 MB                  | 1 GB             | 1 Gbps               |
-| .small                 | 1                        | 2 GB                    | 20 GB            | 1 Gbps               |
-| .medium                | 2                        | 4 GB                    | 40 GB            | 1 Gbps               |
-| .large                 | 4                        | 8 GB                    | 80 GB            | 1 Gbps               |
-| .2xlarge <sup>1)</sup> | 8                        | 16 GB                   | 160 GB           | 1 Gbps               |
-| .4xlarge <sup>1)</sup> | 16                       | 32 GB                   | 320 GB           | 1 Gbps               |
-| .8xlarge <sup>1)</sup> | 32                       | 64 GB                   | 640 GB           | 1 Gbps               |
-
-<p align="center"><b>Table 4-13:</b> Predefined Compute Flavours.</p>
-
-**1)** These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
-**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.<br>
-**3)** The "local disk" is an ephemeral disk that provides storage for the life of a VM/Pod, and can either be provided by storage devices housed within the physical server on which the VM/Pod is running, or from an external storage device such as SAN or NFS.
+
+| Ref | Attribute | Unit | (B)asic/(H)igh Performance | Presence |
+|----------|--------------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------|
+| i.fl.001 | Flavor ID             | integer       | B/H           | Required |
+| i.fl.002 | vCPU                      | integer      | B/H           | Required |
+| i.fl.003 | RAM                       | Gb           | B/H           | Required |
+| i.fl.004 | external_disk             | Gb           | B/H           | Required |
+| i.fl.005 | ephemeral-Disk            | Gb           | B/H           | Optional |
+| i.fl.006 | root-disk                 | GB           | H             | Required
+| i.fl.007 | cpu_SMT                   | Y/N          | H             | Optional
```

Comment:
I think cpu_SMT is enabled for both Basic and High Performance Node Profiles. Given that this will be specified in the Node Profiles it should also be defined in Workload Profile and thus not in Compute Flavour

Reply:
@pgoyal01 In case the host doesn't support SMT Architecture this will fail (hw:cpu_thread_policy set to prefer then it works). Therefore, this shall be used for High Performance

Reply:
@joaoarodrigues This is the Reference Model (RM) and not how and what OpenStack does. The RM is technology agnostic.  Also, while OpenStack states clearly that Flavours are sizing mechanism but then it allows profile information to be included through "extra-specs", etc. As you know SMT is enabled when the infrastructure node is being configured (i.e., per info as in profiles) while flavours are about configuring in the Virtual Infrastructure Layer.